modules/data_team_helper/php/NDB_Form_data_team_helper.class.inc

```diff
@@ -420,6 +420,11 @@ class NDB_Form_Data_Team_Helper extends NDB_Form
         $visit_label=null,
         $candidate=null
     ) {
+        $config         =& NDB_Config::singleton();
+        $ddeInstruments = $config->getSetting('DoubleDataEntryInstruments');
+        for ($i=0; $i<count($ddeInstruments); ++$i) {
```

Comment:
Given count($ddeInstruments) properly returns 0 for an unset/empty array i.e. when no DDE instruments are returned from Config settings, this looks fine.